packages/protocol/contracts/AccountRegistry/epochs/1/base/BehaviourBase1.sol

```diff
@@ -0,0 +1,82 @@
+pragma solidity >=0.5.0 <0.6.0;
+
+import "../../../../libs/LibEIP712.sol";
+
+contract BehaviourBase1 is LibEIP712 {
+    event RegisterExtension(
+        address indexed account,
+        bytes linkedPublicKey,
+        bytes spendingPublicKey 
+    );
+
+    event LogAddress(
+        address account
+    );
+    event LogString(
+        string message
+    );
+
+    event LogBytes(
+        bytes32 sig
+    );
+
+    struct AZTECAccount {
+        address account;
+        bytes linkedPublicKey;
+    }
+
+    mapping(address => bytes) public accountMapping;
+
+    string private constant EIP712_DOMAIN  = "EIP712Domain(string name,string version,address verifyingContract)";
+    string private constant SIGNATURE_TYPE = "AZTECAccount(address account,bytes linkedPublicKey)";
+
+    bytes32 private constant EIP712_DOMAIN_TYPEHASH = keccak256(abi.encodePacked(EIP712_DOMAIN));
+    bytes32 private constant SIGNATURE_TYPEHASH = keccak256(abi.encodePacked(SIGNATURE_TYPE));
+
+
+    /**
+    * @dev function to validate the user is either the sender or has submitted an EIP712 signature from the 
```

Comment:
This function doesn't seem to do any validation.

Reply:
Looks like the docs are out of date with what the function does. Have updated